### ISSUE TITLE
windows 

### ISSUE THREAD

**Issue by SalahNouari** [phrutis/VanitySearch2#2]
please compile this to exe file thanks

**Comment by phrutis** [comment]
What is wrong with the [standard](https://github.com/phrutis/VanitySearch2/releases) assembly?
Do you need a special build of the program?

**Comment by samzy101** [comment]
i get an error saying gpu code not compiled. any ideas as to why?

**Comment by phrutis** [comment]
Take a screenshot of the error

**Comment by samzy101** [comment]
also, just realized its not building correctly. on 2017 vstudio it say i need v142 tools. but thats only available for 2019. and on 2019 if i change the build config to release it just will not compile

**Comment by samzy101** [comment]
![Screenshot (1)](https://user-images.githubusercontent.com/98940834/154355299-43cefbf5-af8c-4972-a3dd-d496935090fd.png)

**Comment by samzy101** [comment]
![Screenshot (2)](https://user-images.githubusercontent.com/98940834/154355483-3a244046-d542-4803-b9df-3ae5e05136ee.png)

**Comment by phrutis** [comment]
I collected on Microsoft Visual Studio Community 2019
Cuda 10.2

**Comment by samzy101** [comment]
I tried all 3 2019 2017 and 2022. For cuda I tried 3 different versions too but never worked

**Comment by samzy101** [comment]
Can you share your compiled version with me?

**Comment by phrutis** [comment]
I gave you a link to the program and asked why you are not satisfied with the standard program? 


>I gave you a link to the program and asked why you are not satisfied with the standard program? 
> What is wrong with the [standard](https://github.com/phrutis/VanitySearch2/releases) assembly? Do you need a special build of the program?

**Comment by Radar-7528** [comment]
Бро! Прива! А где
 от JeanLucPons VanitySearch_v.1.19 для Win, под CUDA 30-той серии.? Или её нет совсем?

**Comment by Radar-7528** [comment]
> Привет. Сам собери. Умеешь?
нет, не умею. 1.19 версия по перфиксу работает, по базе только на процах...
